example/lib/03.change_language_theme/home_screen.dart

```diff
@@ -17,6 +17,7 @@ class HomeScreen extends StatefulWidget {
 class _HomeScreenState extends State<HomeScreen> {
   String _language = _defaultLanguage;
   String _theme = _defaultTheme;
+  final _isSelected = [false, false, false];
```

Comment:
```suggestion
  bool _showNumbers = true;
  bool _showErrors = true;
  bool _showFoldingHandles = true;
```
Then use individual buttons and do not group them in toggle. This is easier to read when learning the editor.